assets/ckeditor_config_basic.js

```diff
@@ -0,0 +1,331 @@
+    //document --> 'NewPage',
+
+    //{ name: 'insert', items: [ 'Image', 'Flash', 'Table', 'HorizontalRule', 'Smiley', 'SpecialChar', 'PageBreak', 'Iframe' ] },
+    //{ name: 'forms', items: [ 'Form', 'Checkbox', 'Radio', 'TextField', 'Textarea', 'Select', 'Button', 'ImageButton', 'HiddenField' ] },
+    //{ name: 'about', items: [ 'About' ] }
+    config.floatSpacePreferRight = true;
+    //config.floatSpaceDockedOffsetX = 300;
+    //config.floatSpaceDockedOffsetY = 100;
+    //config.floatSpacePinnedOffsetX = 1000;
+    ///config.toolbarLocation = 'bottom';
+
+    //Klaas edit -> remove CKeditor red magicline overlay for enter by mouse
+    config.removePlugins = 'magicline';
+    //Klaas edit -> remove CKeditor context menu
+    //config.removePlugins = 'magicline,contextmenu,liststyle,tabletools';
+
+    CKEDITOR.config.magicline_color = '#0000FF';
+
+    //limit Copy Formatting to only allow basic text styles (bold, italic, underline, strikethrough) to be copied:
+    //config.copyFormatting_allowRules = 'b s u i em a strong ul li em p h1 h2 h3; span{text-decoration,font-weight}';
+    config.copyFormatting_disallowRules = 'div';
+
+
+    //configuration to prevent lay-out change onload
+    config.forcePasteAsPlainText = false; // default so content won't be manipulated on load
+    config.basicEntities = true; //Klaas -> do we need this?
+    config.entities = true;
+    config.entities_latin = false;
+    config.entities_greek = false;
+    config.entities_processNumerical = false;
+    config.allowedContent = true; // don't filter my data
+
+
+    //'div,' +
+    config.plugins =
+		'about,' +
+		'a11yhelp,' +
+		'basicstyles,' +
+		'bidi,' +
+		'blockquote,' +
+		'clipboard,' +
+		'colorbutton,' +
+		'colordialog,' +
+		'contextmenu,' +
+		'dialogadvtab,' +
+		'elementspath,' +
+		'enterkey,' +
+		'entities,' +
+		'filebrowser,' +
+		'find,' +
+		'flash,' +
+		'floatingspace,' +
+		'font,' +
+		'format,' +
+		'forms,' +
+		'horizontalrule,' +
+		'htmlwriter,' +
+        'image2,' +
+		'iframe,' +
+		'indentlist,' +
+		'indentblock,' +
+		'justify,' +
+		'language,' +
+		'link,' +
+		'list,' +
+		'liststyle,' +
+        'lineheight,' +
+		'magicline,' +
+		'maximize,' +
+		'newpage,' +
+		'pagebreak,' +
+		'pastefromword,' +
+		'pastetext,' +
+		'preview,' +
+		'print,' +
+		'removeformat,' +
+		'resize,' +
+		'save,' +
+		'selectall,' +
+		'showblocks,' +
+		'showborders,' +
+		'smiley,' +
+		'sourcearea,' +
+		'sourcedialog,' +
+		'stylescombo,' +
+		'tab,' +
+		'table,' +
+		'tabletools,' +
+		'templates,' +
+		'toolbar,' +
+		'undo,' +
+'wysiwygarea';
+	// Remove some buttons provided by the standard plugins, which are
+	// not needed in the Standard(s) toolbar.
+
+    //klaas adapt: extra right-click menu options
+    //config.extraPlugins = 'dialogadvtab';
+
+	//config.removeButtons = 'Underline,Subscript,Superscript';
+
+	// Set the most common block elements.
+    config.format_tags = 'p;h1;h2;h3;pre';
```

Comment:
This has already been changed by Umar's commit https://github.com/slidewiki/slidewiki-platform/commit/b5d875d6d40870cf5ea6fba121ee3bc8b9f07293